lib/notify.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const path = require('path')
+
+module.exports = context => opts => {
+  if (context.flags['no-notify']) return
+  const notifier = require('node-notifier')
+  return notifier.notify({
+    title: `heroku ${process.argv[2]}`,
+    // icon: path.join('Terminal Icon'),
+    icon: path.join(__dirname, '../assets/heroku.png'),
+    // contentImage: path.join(__dirname, '../assets/heroku.png'),
+    // open: // URL to open on Click
```

Comment:
This is for if you click on the notification? Eventually we could add a DHC link, but we can ignore for now.

Reply:
yeah I left all the properties for reference, I figured that one in particular could be useful